languages/JavaScript/exercises/concept/arrays-basic/.docs/after.md

```diff
@@ -0,0 +1,3 @@
+In Javascript, an array is actually just a regular object where the elements are properties of that object. It includes the `length` property and also has lots of useful methods for traversing and mutating the array. More information about the internals of the array [here].
+
```

Comment:
We've recently updated the requirements of the `after.md` file to include examples of any newly introduced syntax. Could you add those to this file?

Reply:
I've also added examples with the `pop`, `push`, `splice `and `length`. Not sure if this is ok, since this is not really language syntax.

Reply:
I think this is _right_. Except that the link is incorrect.

```suggestion
More information about the internals of the array [here](helloooo).
```

Reply:
I don't understand. The link redirect me just fine to [this page](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array).

Reply:
> ovidiu141

I believe you need to specify both the link text and the link, so this should probably be:

```md
More information about the internals of the array [here][arrays].

[arrays]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array
```